### PR TITLE
8 comments associated with article amendments

### PR DESCRIPTION
**Pull request by Daniel-N98**

### CODE REVIEW

File: src/components/Comment.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import "./style/comments.css";
+
+export default function Comment({ comment }) {
+  const dateCreated = new Date(comment.created_at);
+  return (
+    <div className="comment-card flex flex-column article">
```
Comment (P-Copley) [inline src/components/Comment.jsx]:
```js
<div className="comment-card flex flex-column article">
```

As this is a list of comments this should be an `li` and not a `div`

File: src/components/CommentList.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    });
+  }, [article_id]);
+
+  if (isLoading) {
+    return indicateLoading();
+  }
+
+  return (
+    <section id="comments-section" className="text-center">
+      <h1>Comments</h1>
```
Comment (P-Copley) [inline src/components/CommentList.jsx]:
```js
<h1>Comments</h1>
```
This probably isn't the main page title so shouldn't be a `h1`. The heading should descend in order so:
h1 - The single heading of the whole page
h2 - subheadings under that
h3 - subheadings under h2's
etc

File: src/components/CommentList.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      {comments.length > 0 ? (
+        comments.map((comment) => {
+          return <Comment key={comment.comment_id} comment={comment} />;
+        })
+      ) : (
+        <h2>This article has no comments.. be the first?</h2>
```
Comment (P-Copley) [inline src/components/CommentList.jsx]: Same as above ^

**Comment by P-Copley** [comment]
Hey @Daniel-N98 👋 

Lovely functionality here. The semantics of some of your tags are a little off. Use tags that describe the correct meaning of the content and style them to look like what you want afterwards. Otherwise this is looking very good.

Take a look at the semantics I commented on and then merge when you're ready 👍

**Review by popatre — APPROVED** [review]

**Pull request merged**